README.md

```diff
@@ -1,12 +1,17 @@
+[![Netlify Status](https://api.netlify.com/api/v1/badges/3701b349-13d2-40b8-a955-16e595d73fd1/deploy-status)](https://app.netlify.com/sites/ethanzitting-opensgf/deploys)
+
+## Dev Branch Preview
+- [Netlify](https://ethanzitting-opensgf.netlify.app/)
```

Comment:
Don't really think this is necessary to include in the README

Reply:
Sounds good, implemented!